tests/tests.rs

```diff
@@ -1599,6 +1599,18 @@ sherlock!(feature_419_zero_as_shortcut_for_null, "Sherlock", ".",
     assert_eq!(lines, "sherlock\x002\n");
 });
 
+// See: https://github.com/BurntSushi/ripgrep/issues/709
+clean!(suggest_fixed_strings_for_invalid_regex, "foo(", ".",
+|wd: WorkDir, mut cmd: Command| {
+    wd.assert_non_empty_stderr(&mut cmd);
+
+    let output = cmd.output().unwrap();
+    let err = String::from_utf8_lossy(&output.stderr);
+    let expected = "If you wanted to treat the string within double quotes as a literal string, you should pass the --fixed-strings flag.
```

Comment:
Please find a way to wrap this to 79 columns. :)

Also, I'd probably just check that the error message mentions `--fixed-strings`, otherwise the test will probably be unusefully brittle.